.github/workflows/hermetic.yml

```diff
@@ -123,13 +123,13 @@ jobs:
           export TEST_NETWORK=./networks/${{ matrix.networks }}.yaml
           chmod +x ./bin/hermetic-driver
 
-          # For schedule events, "test_selector" will be null, so default to running all tests.
+          # For scheduled events, "test_selector" will be null, so default to running all tests.
           test_selector=${{ github.event.inputs.test_selector || '.' }}
-          # TODO: Uncomment once we have "fast" tests
+
           # For PR and merge group events, run only "fast" tests.
-          # if [[ ${{ github.event_name == 'pull_request' || github.event_name == 'merge_group' }} ]]; then
-          #     test_selector="fast"          
-          # fi
+          if [[ ${{ github.event_name == 'pull_request' || github.event_name == 'merge_group' }} ]]; then
+              test_selector=${{ github.event.inputs.test_selector || 'fast' }}
```

Comment:
I made a small change here vs what you had commented out, because it seemed to me like if the user manually supplies a test selector we should respect it always.  But please let me know if this doesn't seem right for any reason

Reply:
Yes, agreed, should definitely honor user requested selector. The check didn't need to go inside because the manual selection event type is always workflow_dispatch and never pull_request or merge_group.

Reply:
In other words, the original code should still honor the user's selection since it won't be overridden for PR or merge_group events.

Reply:
Updated!